.template-infra/app-analytics.yml

```diff
@@ -0,0 +1,3 @@
+_commit: 929a959
```

Comment:
Not for this PR - but in the template repo, I might recommend a README.md file also be copied over that gives some context on what these files are / what the template repo is.

Just something linking back to that repo and saying "hey we manage a lot of shared configuration that you can pull in, here's the link to the docs".

Reply:
> Not for this PR - but in the template repo

Big agree 👍🏼